examples/performance/laplace/data/BOUT.inp

```diff
@@ -0,0 +1,24 @@
+# Test of the FieldFactory class 
+#
+# Generates a set of fields and writes them to output file
+#
+
+NOUT = 0  # No timesteps
+
+MZ = 32    # Z size
+
+grid = "../../../tests/integrated/test-laplace/test_laplace.grd.nc"
+
+dump_format = "nc"  # NetCDF format. Alternative is "pdb"
+
+[LaplaceTest]
+NUM_LOOPS = 1000
+
+[mesh]
+symmetricGlobalX = false
+symmetricGlobalY = false
+
+[laplace]
+all_terms = false
+include_yguards = false
+filter = 0.2
```

Comment:
Probably don't want a filter in performance tests. It zeros some modes, although I don't remember exactly how. Just deleting the option should be ok, the default now is no filter.

Reply:
I guess it depends what you want to know -- if you're comparing methods this might help point out that some implementations skip work on filtered modes whilst others might not (and others might not support filtering, if we checked the results). Agree default of no filtering probably useful but could perhaps do with a note or similar about this?

Reply:
@bendudson correct me if I'm wrong, but I think we would now recommend never using filtering. So I'd be happy to assume we performance test with the default `filter=0` and if there's someone who really does want filtering and cares about how it affects performance, assume that they're an advanced enough user/developer to modify the test themselves to check.

Reply:
I agree that filtering is probably not a good idea in general, and can make numerical instabilities worse rather than better. There can be a case for filtering in the RHS function (e.g. 2/3rds rule for aliasing), but doing it in the Laplacian is probably not the best way to do it. 
Note: I'd still like to keep it, because there is an exception to (almost) every rule.